portfolio/src/main/webapp/index.html

```diff
@@ -9,7 +9,56 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h2>Introduction</h2>
+      <img src="images/processor.jpg" alt="Fancy image of processor" class="content-container">
+      <p>Hello, I am Joshua Washington. I am a rising junior
+        at Rice University studying computer science and applied mathematics.
+        I am originally from Milwaukee, Wisconsin.
+        My favorite activities include video games, reading, and programming.
+        I am also an avid follower of consumer electronics.
+      </p>
+      <h2>Some Projects I've Worked On</h2>
+      <ul>
+        <li>This one!</li>
+        <li>Calculator that handles complex expressions</li>
+        <li>Text-Based Four-In-a-Line Game</li>
+        <li>Dummy web store</li>
+      </ul>
+      <h2>Favorites</h2>
+
+      <table>
+        <tr>
+          <th>Color</th>
+          <td>Blue</td>
+        </tr>
+        <tr>
+          <th>Book or Book Series</th>
+          <td>(Currently) Lightbringer</td>
+        </tr>
+        <tr>
+          <th>Drink</th>
+          <td>Roast Oolong Iced Tea with Boba</td>
+        </tr>
+        <tr>
+          <th>Video Game</th>
+          <td>Risk of Rain 2</td>
+        </tr>
+        <tr>
+          <th>Board Game</th>
+          <td>Ricochet Robots</td>
+        </tr>
+        <tr>
+          <th>Restaurant</th>
+          <td>Black Walnut Cafe</td>
+        </tr>
+      </table>
```

Comment:
Consider using the description list element here instead of the table element.

This example here shows it used for similar content: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#Metadata

Reply:
Done.